src/applications/vaos/components/AppointmentRequestListItem.jsx

```diff
@@ -63,6 +63,7 @@ export default class AppointmentRequestListItem extends React.Component {
         aria-labelledby={`card-${index} card-${index}-status`}
         data-request-id={appointment.id}
         className={itemClasses}
+        data-is-cancelable={!appointment.isCommunityCare ? 'true' : 'false'}
```

Comment:
This is the item for requests, all requests are cancelable.

Reply:
Yes. The check is needed since cancelling a CC appointment displays a window instructing the user to call the medical center to cancel the appointment. Probable should change this flag to check for VA appointment since video connect appointments display a similar window.